tests/caasp/stack_controller.pm

```diff
@@ -183,9 +182,6 @@ sub run {
         # CaaSP 1.0 = %number_of_jobs - minus three (controller + admin + master) jobs
         $minion_count = get_required_var("STACK_SIZE") - 3;
     }
-    my $temp = get_required_var("STACK_SIZE");
-    assert_script_run "echo $temp";
```

Comment:
JFYI: the reason I added that previous was because of debugging. As a new comer I didn't know exactly how the barriers are working, so I had to make sure of the count of jobs. Now, I am already aware of it, so feel free to remove it ;)

Reply:
yes, I thought that was the reason for $temp code - removed